docker/README.md

```diff
@@ -0,0 +1,108 @@
+# WGDashboard Docker Explanation:
+
+Author: DaanSelen<br>
+
+This document delves into how the WGDashboard Docker container has been built.<br>
+Of course there are two stages, one before run-time and one at/after run-time.<br>
+The `Dockerfile` describes how the container image is made, and the `entrypoint.sh` is executed after running the container. <br>
+In this example, WireGuard is integrated into the container itself, so it should be a run-and-go/out-of-the-box.<br>
+For more details on the source-code specific to this Docker image, refer to the source files, they have lots of comments.
+
+I have tried to embed some new features such as `isolate` and interface startup on container-start (through `enable`). I hope you enjoy!
+
+<img src="https://raw.githubusercontent.com/donaldzou/WGDashboard/main/src/static/img/logo.png" alt="WG-Dashboard Logo" title="WG-Dashboard Logo" width="150" height="150" /> 
+
+## Getting the container running:
+
+To get the container running you either pull the image from the repository, `dselen/wgdashboard:latest`.<br>
+From there either use the environment variables describe below as parameters or use the Docker Compose file: `compose.yaml`.<br>
+Be careful, the default generated WireGuard configuration file uses port 51820/udp. So use this port if you want to use it out of the box.<br>
+Otherwise edit the configuration file in `/etc/wireguard/wg0.conf`.
+
+An example of a simple command to get the container running is show below:<br>
+
+```shell
+docker run -d \
+  --name wireguard-dashboard \
+  --restart unless-stopped \
+  -e enable=wg0 \
+  -e isolate=wg0 \
+  -p 10086:10086/tcp \
+  -p 51820:51820/udp \
+  --cap-add NET_ADMIN \
+  dselen/wgdashboard:latest
+```
+<br>
+If you want to use Compose instead of a raw Docker command, refer to the example in the `compose.yaml` or the one pasted below:
+<br><br>
+
+```yaml
+services:
+  wireguard-dashboard:
+    image: dselen/wgdashboard:latest
+    restart: unless-stopped
+    container_name: wire-dash
+    environment:
+      #- tz=
+      #- global_dns=
+      - enable=none
+      - isolate=wg0
+      #- public_ip=
+    ports:
+      - 10086:10086/tcp
+      - 51820:51820/udp
+    volumes:
+      - conf:/etc/wireguard
+      - app:/opt/wireguarddashboard/app
```

Comment:
This is the wrong mount path.
Needs to be `/opt/wireguarddashboard` according to `Dockerfile` and `compose.yaml` in the alpine folder

Reply:
Its not the wrong PR? He is asking about the current state relating to this image? This image is also fine? Or please inform me about its shortcomings.